### PR TITLE
Node modules

### PR DESCRIPTION
**Pull request by josiahcoad**
- removed spnner and resultsbox
- Using webpack
- Using node modules for tippy and findandreplaceDOMtext
- changed file directory structure
- Using eslint instead of jsbeautify

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Generic Files
 *.txt
 node_modules
-**/contentBundle.js
-**/contentBundle.js.map
\ No newline at end of file
+dist
+!dist/manifest.json
```
Comment (josiahcoad) [inline .gitignore]: do we need to ignore this if we already ignore dist

File: .gitignore [hunk 2]
```diff
@@ -1,5 +1,6 @@
...
+!dist/popup/*
```
Comment (josiahcoad) [inline .gitignore]: whats this with the exclaimation?

File: src/content/index.js [hunk 3]
```diff
@@ -2,10 +2,10 @@
 import findAndReplaceDOMText from 'findandreplacedomtext';
 import {
     getUniqueLocationsFromCurrentPage,
-} from './api';
+} from './api.js';
```
Comment (josiahcoad) [inline src/content/index.js]: was this giving you problems? for me, .js is the default extenstion

File: src/content/index.js [hunk 4]
```diff
@@ -37,6 +37,7 @@ function activatePage() {
 }
 
 // ***************** EXECUTE THIS ON PAGE LOAD ***************** //
+console.log('Activating SeeTheWorld');
```
Comment (josiahcoad) [inline src/content/index.js]: maybe take out before we merge?

File: webpack.config.js [hunk 5]
```diff
@@ -1,18 +1,45 @@
...
-                'css-loader'
-            ]
-        }]
-    }
-};
\ No newline at end of file
+            loader: 'style-loader!css-loader',
+        },
+        {
+            test: /\.html$/,
```
Comment (josiahcoad) [inline webpack.config.js]: whats this? Are we using it?

File: src/content/index.js (hunk 3 above)
Comment (phulsechinmay) [reply src/content/index.js]: It was giving me problems for some reason

File: .gitignore (hunk 2 above)
Comment (phulsechinmay) [reply .gitignore]: The exclamation tells gitignore to _not exclude_ those files / folders.

File: .gitignore (hunk 1 above)
Comment (phulsechinmay) [reply .gitignore]: That is un-ignoring the manifest.json in the dist folder. If we end up having multiple bundle / output files from webpack (One for popup, one for background and more), then we won't need to specifically ignore those files.

Comment (josiahcoad) [reply .gitignore]: ok! thanks

File: .gitignore (hunk 2 above)
Comment (josiahcoad) [reply .gitignore]: ok thanks

File: src/content/index.js (hunk 3 above)
Comment (josiahcoad) [reply src/content/index.js]: oh.. hmm ok well then we can keep it in there for now but eventually lets come back to this becuse i don't believe its common convention

**Pull request merged**